lib/posix/posix.nim

```diff
@@ -93,13 +93,13 @@ else:
 {.deprecated: [cSIG_HOLD: SIG_HOLD].}
 
 when not defined(macosx) and not defined(android):
-  proc st_atime*(s: Stat): Time {.inline.} =
+  proc st_atime*(s: Stat): TimeT {.inline.} =
```

Comment:
This is against our conventions though, it should be ``posix.Time`` vs ``times.Time``. People should disambiguate rather than we having to come up with unique names.